packages/x-components/.eslintrc.js

```diff
@@ -12,21 +12,12 @@ module.exports = {
   },
   overrides: [
     {
-      files: 'src/**/*.vue',
-      rules: {
-        'max-len': [
-          'error',
-          {
-            code: 100,
-            ignoreComments: false,
-            ignorePattern: '|class=".*"$ |https://github'
-          }
-        ]
-      }
-    },
```

Comment:
As we are disabling the entire `max-len` rule for vue components, in order to use` vue/max-len` one, we get rid of this config here.